Gruntfile.js

```diff
@@ -0,0 +1,30 @@
+module.exports = function( grunt ) {
+
+grunt.initConfig( {
+	pkg: grunt.file.readJSON( "package.json" ),
+	jshint: {
+		files: [ "*.js", "routes/**.js" ],
+		options: {
+			jshintrc: ".jshintrc"
+		}
+	},
+	jscs: {
+		src: [ "*.js" ]
```

Comment:
I think this glob will only match js files in the root directory. Should just be all JavaScript files in the repo.